masonry/src/lib.rs

```diff
@@ -1,15 +1,18 @@
 // Copyright 2018 the Xilem Authors and the Druid Authors
 // SPDX-License-Identifier: Apache-2.0
 
-//! A framework that aims to provide the foundation for Rust GUI libraries.
-//!
 //! Masonry gives you a platform to create windows (using [winit] as a backend) each with a tree of widgets. It also gives you tools to inspect that widget tree at runtime, write unit tests on it, and generally have an easier time debugging and maintaining your app.
```

Comment:
I think this first sentence should be rewritten, but that's not required in this PR.